### PR TITLE
Draft - Token transfer

### PR DESCRIPTION
**Pull request by CitMC**
Documentation content update for IBC module, section token transfer

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [ ] Small language/grammar fixes
- [ ] Small content fixes 
- [x] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [ ] completed
- Technical platform review (remove if not required)
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [x] requested
  - [ ] completed
- Language review
  - [x] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [x] requested
  - [x] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [x] Ready to be merged

### CODE REVIEW

File: academy/ibc/token-transfer.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+**Notice:** you can have different channels between two chains, but you cannot transfer the same token across different channels back and forth.
+
+To understand the application logic for a token transfer, first, you have to determine the **source** chain:
+
+![Source chain](images/sourcechain.png)
+
+Then the application logic can be summarized:
+
+![Application logic]
```
Comment (tmsdkeys) [inline academy/ibc/token-transfer.md:19]: insert image

File: academy/ibc/token-transfer.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+    return err
+  }
+
+  return nil
+}
+```
+
+`OnChanOpenAck`, `OnChanOpenConfirm`, `OnChanCloseInit`, and `OnChanCloseConfirm` will do (almost) no checks.
+
+`OnRecvPacket` on the other hand will decode a packet and apply the transfer token application logic:
```
Comment (tmsdkeys) [inline academy/ibc/token-transfer.md:70]: To put it like this, makes it seem as if OnRecvPacket is closely related to the channel handshake steps. Might be worth remembering where in the flow these functions appear

File: academy/ibc/token-transfer.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
+# IBC Fungible Token Transfer
+
+Having looked at the IBC/TAO, you can now take a look at the [ICS-20](https://github.com/cosmos/ibc/blob/master/spec/app/ics-020-fungible-token-transfer/README.md). It describes fungible token transfers. The corresponding [implementation](https://github.com/cosmos/ibc-go/tree/main/modules/apps/transfer) is a module on the application level.
+
+![Overview of a token transfer](images/transferoverview.png)
```
Comment (tmsdkeys) [inline academy/ibc/token-transfer.md:5]: C is not really mentioned in the text?

**Review by tmsdkeys — COMMENTED** [review]
Left some comments to address

File: academy/ibc/token-transfer.md (hunk 3 above)
Comment (CitMC) [reply academy/ibc/token-transfer.md:5]: Will be omitted in the designed image.

**Pull request merged**